lessons/module-3/react-3-advanced-data-management.md

```diff
@@ -1,179 +1,45 @@
 ---
-title: "React: Advanced Data Management"
-length: 180
-tags: react, ideabox, lifecycle methods, conditional rendering
+title: React 3 - Advanced Data Management.
+length: 3 hours
 module: 3
+tags: react, hooks, useEffect, Network request 
 ---
 
 ## Learning Goals
+* Understand how to perform network request and use it within `useEffect`.
```

Comment:
Maybe reword this to be:
"Be able to perform network requests and make use of the response data with the `useEffect` hook"